s32193-SlawomirRudnicki/zadaniaTydzien3.txt

```diff
@@ -66,13 +75,17 @@ zadanie 2
 6. killall bash nie zglosil zadnego bledu, ale nie zaobserwowalem zadnej zmiany
 	przypuszczenie: nic sie nie stalo poniewaz killall nie moze zabic samego siebie
 7. nie zaobserwowalem procesu ktorego nie moglbym zabic uzywajac komendy kill -9
+	POPRAWIENIE: nie, nie bede w stanie zabic procesu innego uzytkownika
+	polaczylem sie z laptopem z maszyny wirtualnej przez SSH, i odpalilem proces timeout, nie moglem go zabic z laptopa 
+	uzywajac kill -9 <PID-procesu> -> Oparation not permitted
 
 zadanie 4
 1. otworzylem gimpa
 2. uzylem komendy ctrl-z aby wstrzymac proces
 3. program "zamarzl", wszelkie proby interakcji koncza sie informacja od systemu ze program nie odpowiada
 4. nie wiem co mialo sie stac, nic wartego adnotacji sie nie wydarzylo
 	okno gimpa moge nadal "przesuwac" mimo ze sam program nie odpowiada
+	ADNOTACJA: zle sformulowalem odpowiedz, po najechaniu innym oknem na wspomnianego gimpa, rowniez nic sie nie wydarzylo
```

Comment:
I po odsunięciu tego okna przysłaniającego wstrzymanego gimpa, okno gimpa się wyrysowało bez mozaiki?
Hmm... to może jakaś aktualizacja w serwerze graficznym na przestrzeni lat nastąpiła.